### ISSUE TITLE
Daten für die Wohnungsmarktbeobachtung

### ISSUE THREAD

**Issue by OpenDirk** [codefordus/Wohnungsmarktbeobachtung#1]
Die Daten, die jetzt als Excel-Dateien von der Statistik bereit stehen, werden in Kürze als csv-Dateien im Open-Data-Portal zur Verfügung sein.
Wenn jemand schon vorher was braucht, soll er/sie sich bitte bei mir melden

Dirk

**Comment by t7l** [comment]
Das klingt sehr gut, vielen Dank.

**Comment by OpenDirk** [comment]
Ich brauche mal eine Einführung in den Umgang mit GitHub - können wir das beim nächsten OK-Lab machen?

**Comment by t7l** [comment]
Ja, ich kann dazu gerne einen kleinen Input und Beispiele beisteuern.
Ansonsten ist das interaktive Tutorial schon mal ein netter Anfang: https://try.github.io
Ich kann dich der Organisation und den Team nicht zuordnen. Hast Du das evtl. in den Einstellungen geblockt?

**Comment by t7l** [comment]
Gibt es dann auch Daten zu den IDs der Wohnquartiere? Ich habe mir die Excel-Dateien mal angesehen. Für die Umsetzung beispielsweise auf einer Karte wären dann noch Polygone o.ä. nötig, mit welchen die Wohnquartiere abgebildetb werden können. Auf den Seiten zu den veröffentlichten Daten und auch bei weiteren Recherchen konnte ich auf Anhieb nichts dazu finden.

**Comment by OpenDirk** [comment]
Die räumliche Gliederung habe ich als GEOJSON und KML aufbereitet. Die Dateien kommen auch ins Portal - brauchst du vorher was?

**Comment by t7l** [comment]
Wenn es keinen Aufwand macht, könnte ich mir mit GeoJSON und KML schon mal Verarbeitungsmöglichkeiten für Karten angucken. Ansonsten reicht es dann auch im Portal.

**Comment by OpenDirk** [comment]
das Einladen sollte jetzt funktionieren

**Comment by OpenDirk** [comment]
Jetzt sind alle Daten der Wohnungsmarktbeobachtung im Portal verfügbar
Quasi als Weihnachtsgeschenk :)
